lms/djangoapps/certificates/queue.py

```diff
@@ -214,6 +214,18 @@ def add_cert(self, student, course_id, course=None, forced_grade=None, template_
         Returns the newly created certificate instance
```

Comment:
Could you please add comments about CCX and your change to the comment block before this line?